docs/core/tools/dotnet-new-update.md

```diff
@@ -9,49 +9,81 @@ ms.date: 04/29/2021
 
 ## Name
 
-`dotnet new --update-check` - Checks for available updates for installed template packages.
-
-`dotnet new --update-apply` - Applies updates to installed template packages.
+`dotnet new update` - updates installed template packages.
+`dotnet new update --check-only` - Checks for available updates for installed template packages without applying them.
 
 ## Synopsis
 
 ```dotnetcli
-dotnet new --update-check
+dotnet new update [--interactive] [--add-source|--nuget-source <SOURCE>] 
+    [-d|--diagnostics] [--verbosity <LEVEL>] [-h|--help]
 
-dotnet new --update-apply
+dotnet new update --check-only|--dry-run [--interactive] [--add-source|--nuget-source <SOURCE>] 
+    [-d|--diagnostics] [--verbosity <LEVEL>] [-h|--help]
 ```
 
 ## Description
 
-The `dotnet new --update-check` option checks if there are updates available for the template packages that are currently installed.
-The `dotnet new --update-apply` option checks if there are updates available for the template packages that are currently installed and installs them.
+The `dotnet new update` command updates installed template packages.
+The `dotnet new update` command with `--check-only` option checks for available updates for installed template packages without applying them.
```

Comment:
Should those two sentences be as well separated to two separate lines? (extra spaces at the ned of lline needed for that - see previous comment)

Reply:
It looks ok to me. I would prefer maybe 
```suggestion
The `dotnet new update` command updates installed template packages. The command with `--check-only` option checks for available updates for installed template packages without applying them.
```
to avoid duplication.